1.0.0 doc changes

cs-abhishek-shukla opened the pull request:
- Fixed Typo, Grammar, label names, doc structure, and information flow issues

cs-suyog-jinturkar commented on docs/contents.md:
```diff
@@ -8,90 +8,84 @@ The **Outbreak Response Framework** solution pack contains the following resourc
-| Record Status |
+| Name                    | Description |
+|:------------------------|-------------|
+| AlertType               |             |
```
Sujit needs to add desc for this artefacts.

cs-abhishek-shukla replied:
yes. Awaiting inputs

cs-suyog-jinturkar commented on docs/setup.md:
```diff
@@ -11,82 +11,95 @@
     ![Outbreak Alert get started](./res/config-wizard-01.png)
 
-3. Specify Threat Detection Integrations sources to run outbreak response hunt activities. Click **Next** to proceed.
+4. Select Threat Detection Integration sources to run outbreak response hunt activities and click **Next**.
```
Why its needed or recommended can be added here.

cs-abhishek-shukla replied:
Added

cs-suyog-jinturkar commented on docs/setup.md:
```diff
@@ -11,82 +11,95 @@
     ![Define your Threat Detection Integrations](./res/config-wizard-02.png)
 
-4. Specify Ticketing/ITSM Integrations sources as part of your response or threat management strategy. Click **Next** to proceed.
+5. Select Ticketing/ITSM Integration sources as part of your response or threat management strategy and click **Next**.
```
same a above. Intent of having this option is important here.

cs-abhishek-shukla replied:
Done

cs-suyog-jinturkar commented on docs/setup.md:
```diff
@@ -11,82 +11,95 @@
-- Fortinet FortiSIEM Config: 
-![Fortinet FortiSIEM Config](./res/fsm_config.png)
+2. Open the **Investigate Outbreak** playbook from the *10 - SP - Outbreak Response Framework* playbook collection.
+Add the following configuration parameters in the *Set Variable* step depending on the *Threat Detection Integrations* that you have selected in the **Configuration** wizard.
```
It will be good if we also explain them here. Any thoughts why these are not described here?

cs-suyog-jinturkar commented on docs/setup.md:
```diff
@@ -11,82 +11,95 @@
-3. If you want to add "Ticketing/ITSM Integrations" then added "Ticketing/ITSM Integrations" name "Ticketing Tools" picklist.
-4. Run "Outbreak Response Framework" configuration wizard and select the old and new added integration either in "Threat Detection Integrations" or "Ticketing/ITSM Integrations" 
-5. Following global variables must never be modified or deleted:
+To add new *Threat Detection Integrations* or *Ticketing/ITSM Integrations* apart from the ones listed in the **Configuration Wizard**, consider the following pointers:
```
This can be treated as advance settings and not best practices as it not applicable for all.

cs-abhishek-shukla replied:
Done.

cs-dev-chaudhary commented on docs/setup.md:
```diff
@@ -1,92 +1,156 @@
-| Vulnerability Management     | Solution Pack     | v1.2.1  | Required to ingest CVE's for KEV's |
+| Name                     | Type          | Version | Purpose                                |
+|:-------------------------|:--------------|:--------|:---------------------------------------|
+| SOAR Framework           | Solution Pack | v2.2.0  | Required for Incident Response modules |
```
SFSP version can be removed. We support earlier version as well. 
https://mantis.fortinet.com/bug_view_page.php?bug_id=0961045

cs-dev-chaudhary commented on docs/setup.md:
```diff
@@ -1,92 +1,156 @@
+| SOAR Framework           | Solution Pack | v2.2.0  | Required for Incident Response modules |
+| Threat Intel Management  | Solution Pack | v1.1.0  | Required to ingest threat feeds        |
+| Vulnerability Management | Solution Pack | v1.2.1  | Required to ingest CVE's for KEV's     |
 
```
Need to add "create ticket in ITSM tool" playbook needs to be activated in prerequisite. https://mantis.fortinet.com/bug_view_page.php?bug_id=0961072

cs-abhishek-shukla replied:
Done

cs-dev-chaudhary commented on docs/setup.md:
```diff
@@ -1,92 +1,156 @@
+- **ServiceNow** - ServiceNow connector provides functionality to create, read, update, and delete records of type Table and Catalog. To configure and use the ServiceNow connector, refer to [Configuring ServiceNow](https://docs.fortinet.com/fortisoar/connectors/servicenow-v2-0-1)
+- **Splunk** - Splunk connector allows users to invoke search, fetch events to related search, invoke alert actions, update notables, sync Splunk users to FortiSOAR, etc. To configure and use the Splunk connector, refer to [Configuring Splunk](https://docs.fortinet.com/fortisoar/connectors/splunk_new)
+
+- **ElasticSearch** - ElasticSearch is a distributed, RESTful search and analytics engine capable of solving several use cases. At the heart of the Elastic Stack, it centrally stores your data so you can discover the expected and uncover the unexpected. To configure and use the ElasticSearch connector, refer to [Configuring ElasticSearch](https://docs.fortinet.com/fortisoar/connectors/)
```
Elastic search is removed. We need to take this out.

cs-abhishek-shukla replied:
Done

cs-dev-chaudhary commented on docs/setup.md:
```diff
@@ -1,92 +1,156 @@
+
+    1. For **Fortinet FortiSIEM** configuration create and specify values for following variables:
+
+        - `faz_ADOM`
```
These are fortianalyzer variables, we need to move the variable for all the SIEM   in the document.

cs-abhishek-shukla replied:
Done. Please check for FGT variable in FAZ

cs-dev-chaudhary commented on docs/setup.md:
```diff
@@ -1,92 +1,156 @@
 
-- Fortinet FortiSIEM Config: 
-![Fortinet FortiSIEM Config](./res/fsm_config.png)
+        - `fgt_policy_name`
```
These are fortigate variables

cs-abhishek-shukla replied:
Done.

cs-dev-chaudhary commented on docs/setup.md:
```diff
@@ -1,92 +1,156 @@
 
-- Splunk Config:
-![Splunk Config](./res/splunk_config.png)
+        - `fsm_event_count`
```
Fortisiem variables.

cs-abhishek-shukla replied:
Done

cs-dev-chaudhary commented on docs/setup.md:
```diff
@@ -1,92 +1,156 @@
+
+    3. For **Fortinet FortiGate** configuration create and specify values for following variables:
+
+        - `gradar_max_events`
```
Qradar variable

cs-abhishek-shukla replied:
Done.

cs-dev-chaudhary commented on docs/setup.md:
```diff
@@ -1,92 +1,156 @@
+
+    3. For **Fortinet FortiGate** configuration create and specify values for following variables:
+
+        - `splunk_index`
```
splunk variables.

cs-abhishek-shukla replied:
Done.

cs-dev-chaudhary commented on docs/contents.md:
```diff
@@ -8,90 +8,84 @@ The **Outbreak Response Framework** solution pack contains the following resourc
-| ElasticSearch | ElasticSearch is a distributed, RESTful search and analytics engine capable of solving a number of use cases. This connector facilitates automated operations to execute lucene query, get mapping and cluster details. |
+| ServiceNow                           | ServiceNow connector provides functionality to create, read, update and delete records of Table and Catalog type                                                                                                                                                                                                                                                       ...
+| Splunk                               | Splunk connector allows users to invoke search, fetch events to related search, invoke alert actions, update notables, sync splunk users to FortiSOAR etc.                                                                                                                                                                                                             ...
+| ElasticSearch                        | ElasticSearch is a distributed, RESTful search and analytics engine capable of solving a number of use cases. This connector facilitates automated operations to execute Lucene query, get mapping and cluster details.                                                                                                                                                ...
```
Elastic search needs to be removed.

cs-abhishek-shukla replied:
Done.

cs-sujeet-ahirrao commented:
@cs-abhishek-shukla  Please add the dashboard screenshot in documentation and related information (The widget information itself explains its usage). If you need any help please reach-out to @cs-dev-chaudhary

cs-dev-chaudhary commented on docs/usage.md:
```diff
@@ -1,62 +1,73 @@
+5. Ingest IOCs as Threat Feeds: IOCs associated with the Outbreak are ingested as threat feeds in FortiSOAR.
 
-5. Ingest IOCs as Threat Feeds: IOCs associated to the Outbreak are ingested as threat feeds in FortiSOAR Say, a new IOC is added to the same outbreak, user does not need to update the solution pack, as we will be hosting the IOCs on our public github page, where our team will be updating them. so threat hunting queries always get the latest IOCs.
+    IOCs are hosted on our public GitHub page and regularly updated by the FortiSOAR team so the threat hunting queries have the latest IOCs. These regular updates avoid the need to update the solution pack if a new IOC is added to the same outbreak.
```
We are no longer fetching the IOCs from github. Instead we are using Fortiguard API to fetch IOCs.

cs-abhishek-shukla replied:
Fixed.

cs-dev-chaudhary commented on docs/usage.md:
```diff
@@ -1,62 +1,73 @@
-    a. Perform IOC Threat Hunting using Fortinet Fabric solutions (FortiSIEM/FortiAnalyzer) and creates Outbreak IOC Hunt Alerts in FortiSOAR
+    a. Fortinet Fabric solutions (FortiSIEM/FortiAnalyzer)
 
-    b. Perform IOC Threat Hunting using SIEM solutions (QRadar/Splunk) and creates Outbreak IOC Hunt Alerts in FortiSOAR
```
We can add fortisiem and fortianalyzer too under SIEM solutions.

cs-abhishek-shukla replied:
Fixed

cs-dev-chaudhary commented on docs/usage.md:
```diff
@@ -1,62 +1,73 @@
 
     ![FortiAnalyzer Fabric Rule](./res/faz_fortinet_fabric.png)
 
-    b. Perform Signature Based Threat Hunting using SIEM solutions (QRadar/Splunk/Azure Log Analytics) and creates Outbreak Alerts in FortiSOAR.
```
Create a alert instead of "create Outbreak Alerts"

cs-dev-chaudhary commented on docs/usage.md:
```diff
@@ -1,62 +1,73 @@
     ![FortiAnalyzer Fabric Rule](./res/faz_fortinet_fabric.png)
 
-    b. Perform Signature Based Threat Hunting using SIEM solutions (QRadar/Splunk/Azure Log Analytics) and creates Outbreak Alerts in FortiSOAR.
+    b. Perform Signature-based Threat Hunting using SIEM solutions (QRadar/Splunk/Azure Log Analytics) and create Outbreak Alerts in FortiSOAR.
```
Foritisiem and fortianalyzer can also be added.

cs-abhishek-shukla replied:
Fixed

cs-dev-chaudhary commented on README.md:
@cs-abhishek-shukla we need to add preview tag and note. Just FYI.

cs-abhishek-shukla committed:
1.0.0 doc changes
- Fixed Typo, Grammar, label names, doc structure, and information flow issues
- Added workaround
- Added example
- Created known issue, inactive, and preview tags
- Rewrote usage and setup
